PayPalUI/src/main/java/com/paypal/android/ui/paymentbutton/PayLaterButton.kt

```diff
@@ -40,13 +39,16 @@ class PayLaterButton @JvmOverloads constructor(
             }
         }
 
-    override val wordmarkDarkLuminanceResId: Int = R.drawable.logo_paypal_monochrome
+    override val wordmarkDarkLuminanceResId: Int
+        get() = R.drawable.logo_paypal_monochrome
 
-    override val wordmarkLightLuminanceResId: Int = R.drawable.logo_paypal_color
+    override val wordmarkLightLuminanceResId: Int
+        get() = R.drawable.logo_paypal_color
 
-    override val fundingType: PaymentButtonFundingType = PaymentButtonFundingType.PAY_LATER
+    override val fundingType: PaymentButtonFundingType
+        get() = PaymentButtonFundingType.PAY_LATER
```

Comment:
@kgangineni i think here is a merge conflict issue. I fixed this on the preivous PR, removing the `get()` accesors and simply assign it to the variable. Can you revert this change?